src/v/cluster/partition_balancer_backend.cc

```diff
@@ -358,9 +362,22 @@ ss::future<> partition_balancer_backend::do_tick() {
     // status requests by default 700ms
     const auto node_responsiveness_timeout = _node_status_interval() * 7;
 
+    const bool should_sanction = _feature_table.should_sanction();
+
+    const auto [mode, is_sanctioned] = _mode(should_sanction);
+    if (is_sanctioned) {
+        vlog(
```

Comment:
We wanted to rate limit logging these, right? Looks like this is the way to do it:
https://github.com/redpanda-data/redpanda/blob/6c81b34ef54a5dceea834b4d0411d0344e4d7e83/src/v/cluster/node_status_backend.cc#L251-L265

Reply:
We never reached a consensus here.
@michael-redpanda what should we do? Should it be limited? (And to what)

Reply:
Closing the loop here that we've agreed that we don't need to rate limit here because we expect the tick period to be >1s based on the fact that the default is 30s.